components/productions/index/event.vue

```diff
@@ -126,4 +126,10 @@
       display: none;
     }
   }
+
+  .table {
```

Comment:
Nit: The `.table` class is a bit redundant though, no? 😛 But I know it was already there in the template.